examples/tdigest.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Contributors to the Cellarium project.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Example: One-pass calculation of feature median using t-digest statistics
+=========================================================================
+
+This example shows how to calculate non-zero median of log normalized feature count
+data in one pass [1].
+
+Example run::
+    python examples/tdigest.py fit \
```

Comment:
In the future, this should also take the normalization config YAML.  We would bootstrap:  first, we run tdigest with a basic normalization config YAML that disables scaling by median and z-scoring (and provides no path to those checkpoints), but optionally enables log1p and/or library size normalization.  We then use the obtained checkpoint to make a new normalization config YAML.  We should check that tdigest checkpoint basic normalization config YAML is compatible with the new normalization config YAML.  For example, if we log1p before tdigest and making the checkpoint, the new normalization config YAML must also enable log1p, etc.